src/fec_context.h

```diff
@@ -140,17 +146,12 @@ class DecodeContext {
         }
     }
 
-    void init(
-        vec::Vector<T>& fragments_ids,
-        const vec::Vector<T>& vx,
-        bool nf4 = false)
+    void init(const vec::Vector<T>& vx)
```

Comment:
Can this one be private?
I mean, what is it's use case (now that the init is fully handled by the constructor)?